service/src/main/java/com/fecklessweasel/service/UniversityServlet.java

```diff
@@ -19,12 +19,13 @@
 	/**
 	 * Handles post requests to this end point. Performs the creation
 	 * of universities given a set of parameters.
-	 * **/
+	 **/
 	String longName;
```

Comment:
Why are all variables defined out here as fields? This is expanding their scope beyond the function to the class as a whole and is a dangerous loss of encapsulation. Define variables *in* function, not in class unless they are part of the object.